test/test_package.py

```diff
@@ -191,7 +192,7 @@ def test_save_imported_module_fails(self):
 
         f2 = self.temp()
         pe = PackageExporter(f2, verbose=False)
-        pe.importers.insert(0, importer1.import_module)
+        pe.module_env = ModuleEnv([importer1, DefaultImporter])
```

Comment:
I don't like that users have to now know where to find DefaultImporter and if it is forgotten, then pickling is going to almost always fail (can't save things like builtins, etc.). This also doesn't let different parts of the exporter script register importers but requires that they are specified upfront when constructing ModuleEnv. Maybe we should keep `pe.importers` and just create a `ModuleEnv` in a @property on PackageExporter when requested.

Reply:
I feel like it's good to be explicit and simple, even if a little verbose, given how strange the behavior is likely to be when things are wrong. Like, I feel like users should know that DefaultImporter is in there—if they forgot, their code is probably going to not work anyway (since doing `.append()` to an "empty" list is more natural than inserting at 0).